dist/index.d.ts

```diff
@@ -1,5 +1,5 @@
 import type { GetStarknetWalletOptions, IStarknetWindowObject } from "./types";
-export declare const getStarknet: () => IStarknetWindowObject;
+export declare const getStarknet: (resetAuthorizations?: boolean | undefined) => IStarknetWindowObject;
 export declare const connect: (options?: GetStarknetWalletOptions | undefined) => Promise<IStarknetWindowObject | undefined>;
-export declare const disconnect: () => boolean;
+export declare const disconnect: (resetAuthorizations?: boolean | undefined) => boolean;
```

Comment:
```suggestion
export declare const disconnect: (resetAuthorizations?: boolean) => boolean;
```
not needed, as it's optional anyways (`?`)

Reply:
Seems to have been added by `npm run build`.

Reply:
oh! build files will (hopefully) be excluded soon anyways! :D